src/io/react-native/react-native-worker.ts

```diff
@@ -196,12 +200,12 @@ async function makeIo(clientIo: ClientIo): Promise<EdgeIo> {
         try {
           const response = await fetchCorsProxy(uri, opts)
           if (response.status === 418) {
-            throw new Error(PROXY_ERROR)
+            throw new NetworkError()
           }
           hostnameProxyWhitelist.add(hostname)
           return response
         } catch (error) {
-          if (errorToThrow == null && String(error) !== PROXY_ERROR)
+          if (errorToThrow == null && asMaybeNetworkError(error) != null)
```

Comment:
Nitpick, I think this should be:

asMaybeNetworkError(error) == null)

because the error we ultimately throw should originate from a fetch function and not our artificial teapot error

Reply:
Derp.